samples/nrf9160/modem_shell/src/location/location_shell.c

```diff
@@ -115,8 +112,10 @@ enum {
 	LOCATION_SHELL_OPT_GNSS_LOC_CLOUD_PVT,
 	LOCATION_SHELL_OPT_CELLULAR_TIMEOUT,
 	LOCATION_SHELL_OPT_CELLULAR_SERVICE,
+	LOCATION_SHELL_OPT_CELLULAR_RESP_DISABLED,
```

Comment:
Please, put _EXT_ to higlight that thsese are for ext architecture.

Reply:
I'll add it then throughout (also command line option name and variables) this file.
The correct name for this would be `LOCATION_SHELL_OPT_SERVICE_EXT_CELLULAR_RESP_DISABLED` or `LOCATION_SHELL_OPT_CELLULAR_SERVICE_EXT_RESP_DISABLED` but I guess `LOCATION_SHELL_OPT_CELLULAR_EXT_RESP_DISABLED` is more reasonable in length.

Are we OK with command line option `--cellular_ext_resp_disabled`?

Reply:
I'm not sure we need to have "ext" also in the option name. Especially if you do what @jhirsi suggested, i.e. flag the option with `CONFIG_LOCATION_SERVICE_EXTERNAL`.

Reply:
ok, I'll leave the option name untouched.

Reply:
Because we didn't want to add "ext" into command line argument to keep it shorter and leave the option of using it for nRF cloud (or some other cloud solution) in the future, I didn't add it to these enums or variables either to be consistent.